# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pandas\core\reshape\melt.py
# hypothesis_version: 6.140.2

['DataFrame.melt', 'F', '\\d+', 'caller', 'id_vars', 'melt', 'other', 'pd.melt(df, ', 'value', 'value_vars', 'variable']